hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -189,7 +189,7 @@
     CONSUMER_RATE_CONVERGENCE_FACTOR("consumer.rate.convergence.factor", 0.2),
     CONSUMER_RATE_FAILURES_NOCHANGE_TOLERANCE_RATIO("consumer.rate.failures.nochange.tolerance.ratio", 0.05),
     CONSUMER_RATE_FAILURES_SPEEDUP_TOLERANCE_RATIO("consumer.rate.failures.speedup.tolerance.ratio", 0.01),
-    CONSUMER_MAXRATE_STRATEGY("consumer.maxrate.strategy", "strict"),
+    CONSUMER_MAXRATE_STRATEGY("consumer.maxrate.strategy", "negotiated"),
```

Comment:
Please see my comment on `StrictMaxRateProvider` and the `@Deprecated` annotation.